src/main/java/command/DoneCommand.java

```diff
@@ -0,0 +1,33 @@
+package command;
+
+import exception.InvalidValue;
+import storage.Storage;
+import task.TaskList;
+import ui.Ui;
+
+/**
+ * DoneCommand will mark as task as completed when executed.
+ */
+public class DoneCommand extends Command {
+
+    private int taskNumber;
+    private final boolean EXIT = false;

```

Comment:
I like how you make `EXIT` as `private final` and always set to `false` for any command that is not `exit`